File: problem-1.py
# -*- coding: utf-8 -*-
"""
Created on Tue Nov 20 09:42:37 2018

@author: Adri Jovin
"""
limit=input("Enter the limit:")
sum=0
for i in range(int(limit)):
    if i%3==0 or i%5==0:
        sum=sum+i
print(sum)

File: problem-1.c
#include<stdio.h>
int main()
{

    int limit, i, sum;
    sum=0;
    printf("Enter the limit:");
    scanf("%d", &limit);
    for(i = 1; i < limit; i++)
    {
        if((i%3==0)||(i%5==0))
        {
            sum=sum+i;
        }
    }
    printf("\nThe result is %d",sum);
    return 0;
}
